src/System.Runtime.Numerics/src/System.Runtime.Numerics.csproj

```diff
@@ -19,7 +19,10 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System\Numerics\BigIntegerCalculator.AddSub.cs" />
+    <Compile Include="System\Numerics\BigIntegerCalculator.Buffer.cs" />
     <Compile Include="System\Numerics\BigIntegerCalculator.DivRem.cs" />
+    <Compile Include="System\Numerics\BigIntegerCalculator.PowMod.cs" />
+    <Compile Include="System\Numerics\BigIntegerCalculator.Reducer.cs" />
     <Compile Include="System\Numerics\BigIntegerCalculator.SquMul.cs" />
     <Compile Include="System\Numerics\BigInteger.cs" />
     <Compile Include="System\Numerics\BigIntegerBuilder.cs" />
```

Comment:
I'd like to port the remaining code to BigIntegerCalculator, then we could drop it. If that's okay, I'd prefer a separate PR, since it will take some time.